auto_submit/lib/service/revert_review_template.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// A template for creating follow up review issues for created revert requests.
+class RevertReviewTemplate {
+  RevertReviewTemplate({
+    required this.repositorySlug,
+    required this.revertPrNumber,
+    required this.revertPrAuthor,
+    required this.originalPrLink,
+  });
+
+  final String repositorySlug;
+  final int revertPrNumber;
+  final String revertPrAuthor;
+  final String originalPrLink;
+
+  /// Constructs the issues title.
+  String _constructTitle() {
+    return '''
+Review request for Revert PR $repositorySlug#$revertPrNumber
+''';
+  }
+
+  /// Constructs the issues body.
+  String _constructBody() {
+    return '''
+Pull request $repositorySlug#$revertPrNumber was submitted and merged by
+$revertPrAuthor in order to revert changes made in this pull request $originalPrLink.
```

Comment:
Appending `@` so that the people will be auto pinged?

Reply:
Yeah mean in from of the revertPrAuthor?

Reply:
Right.